perception/compare_map_segmentation/include/compare_map_segmentation/voxel_grid_map_loader.hpp

```diff
@@ -0,0 +1,224 @@
+// Copyright 2023 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COMPARE_MAP_SEGMENTATION__VOXEL_GRID_MAP_LOADER_HPP_
+#define COMPARE_MAP_SEGMENTATION__VOXEL_GRID_MAP_LOADER_HPP_
+
+#include "compare_map_segmentation/multi_voxel_grid_map_update.hpp"
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <autoware_adapi_v1_msgs/msg/localization_initialization_state.hpp>
```

Comment:
Where is this used? :pray:

Reply:
I'm sorry, it is typo. I fixed [here](https://github.com/autowarefoundation/autoware.universe/pull/3087/commits/f74ee8feeeb76e9daddda1867eca3eecdc7b929a)